README.yaml

```diff
@@ -9,43 +9,44 @@ badges:
     Security Group for EKS Workers
   url: https://github.com/cloudposse/terraform-aws-eks-workers
-- name: terraform-aws-ec2-autoscale-group
-  description: Terraform module to provision Auto Scaling Group and Launch Template
-    on AWS
-  url: https://github.com/cloudposse/terraform-aws-ec2-autoscale-group
-- name: terraform-aws-ecs-container-definition
-  description: Terraform module to generate well-formed JSON documents (container
-    definitions) that are passed to the  aws_ecs_task_definition Terraform resource
-  url: https://github.com/cloudposse/terraform-aws-ecs-container-definition
-- name: terraform-aws-ecs-alb-service-task
-  description: Terraform module which implements an ECS service which exposes a web
-    service via ALB
-  url: https://github.com/cloudposse/terraform-aws-ecs-alb-service-task
-- name: terraform-aws-ecs-web-app
-  description: Terraform module that implements a web app on ECS and supports autoscaling,
-    CI/CD, monitoring, ALB integration, and much more
-  url: https://github.com/cloudposse/terraform-aws-ecs-web-app
-- name: terraform-aws-ecs-codepipeline
-  description: Terraform module for CI/CD with AWS Code Pipeline and Code Build for
-    ECS
-  url: https://github.com/cloudposse/terraform-aws-ecs-codepipeline
-- name: terraform-aws-ecs-cloudwatch-autoscaling
-  description: Terraform module to autoscale ECS Service based on CloudWatch metrics
-  url: https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-autoscaling
-- name: terraform-aws-ecs-cloudwatch-sns-alarms
-  description: Terraform module to create CloudWatch Alarms on ECS Service level metrics
-  url: https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-sns-alarms
-- name: terraform-aws-ec2-instance
-  description: Terraform module for providing a general purpose EC2 instance
-  url: https://github.com/cloudposse/terraform-aws-ec2-instance
-- name: terraform-aws-ec2-instance-group
-  description: Terraform module for provisioning multiple general purpose EC2 hosts
-    for stateful applications
-  url: https://github.com/cloudposse/terraform-aws-ec2-instance-group
+- name: terraform-aws-eks-fargate-profile
+  description: Terraform module to provision an EKS Fargate Profile
+  url: https://github.com/cloudposse/terraform-aws-eks-fargate-profile
+- name: terraform-aws-eks-node-group
+  description: Terraform module to provision an EKS Node Group
+  url: https://github.com/cloudposse/terraform-aws-eks-node-group
+- name: terraform-aws-eks-iam-role
+  description: Terraform module to provision an EKS IAM Role for Service Account
+  url: https://github.com/cloudposse/terraform-aws-eks-iam-role
+- name: terraform-aws-eks-spotinst-ocean-nodepool
+  description: A terraform module to create an EKS node pool with Spot Ocean by Spot.io
+  url: https://github.com/cloudposse/terraform-aws-eks-spotinst-ocean-nodepool
+- name: terraform-provider-aws
+  description: terraform-provider-aws
```

Comment:
Also, can you use the description from github?

![image](https://user-images.githubusercontent.com/52489/105900609-247e6980-5fd1-11eb-988c-19fa0cfbc31f.png)